orchestrator/careplanservice/handle_updatetask.go

```diff
@@ -32,6 +32,16 @@ func (s *Service) handleUpdateTask(httpRequest *http.Request, tx *coolfhir.Trans
 		return nil, fmt.Errorf("failed to read Task: %w", err)
 	}
 
+	// Validate fields on updated Task
+	// if Intent is not set on the updated task then just get it from the existing task, otherwise run usual validation rules
+	if task.Intent == "" {
+		task.Intent = taskExisting.Intent
```

Comment:
why default, and not return an error?

Reply:
I figured it might be annoying if you didn't set the field but you didn't want it to change, that being said though I realise this is a PUT and not a PATCH and my reasoning doesn't make sense in that case. I will remove this and let the existing validation handle it